### ISSUE TITLE
Write chapter: Collaborating on GitHub/GitLab

### ISSUE THREAD

**Issue by pherterich** [alan-turing-institute/the-turing-way#327]
## Detailed description

This issue is to collect ideas and organise the writing on a chapter on "Collaborating on GitHub/GitLab". The chapter will mainly cover Mozilla WOW material from http://mozillascience.github.io/working-open-workshop/

### Who can help?

Everyone with ideas what to include, useful links and especially volunteers to write sections :sparkle: 

## Current status

This chapter is fairly empty, a basic start is made at https://hackmd.io/8VUrkkBeR8iu0SHzucjQog

- [ ] Add your ideas to this issue
- [ ] Write chapter outline
- [ ] Add material to the chapter
- [ ] Combine materials into a readable chapter
- [ ] Proofread
- [ ] Request reviews
- [ ] Address reviews
- [ ] Merge to master branch.

**Comment by rainsworth** [comment]
The Open Science MOOC also has some useful material on this topic:
* https://github.com/OpenScienceMOOC/Module-5-Open-Research-Software-and-Open-Source
* https://eliademy.com/catalog/oer/module-5-open-research-software-and-open-source.html

such as the practicalities of getting started with GitHub, making a repo citable with zenodo, etc.

**Comment by pherterich** [comment]
Forgot to link this to my PR back then - should we close this and open an issue on all the improvements that could go in there? @rainsworth @KirstieJane - could you summarize in a new issue what you feel the chapter is lacking at the moment?

**Comment by pherterich** [comment]
Just realized that #460 is already around as a follow up chapter, so I will close this one and hope everything else gets picked up in future issues.
